.dependabot/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 1
+update_configs:
+  - package_manager: "ruby:bundler"
+    directory: "/"
+    update_schedule: "weekly"
+    default_reviewers:
+      - "who should go here?"
```

Comment:
@larouxn This default reviewers can be 

1. sony-mathew
2. sojan-official
3. subintp